src/docbuilder/rustwide_builder.rs

```diff
@@ -108,6 +118,23 @@ impl RustwideBuilder {
     }
 
     pub fn update_toolchain(&mut self) -> Result<bool> {
+        // For CI builds, a lot of the normal update_toolchain things don't apply.
+        // CI builds are only for one platform (https://forge.rust-lang.org/infra/docs/rustc-ci.html#try-builds)
+        // so we only try installing for the current platform. If that's not a match,
+        // for instance if we're running on macOS or Windows, this will error.
+        // Also, detecting the rustc version relies on calling rustc through rustup with the
+        // +channel argument, but the +channel argument doesn't work for CI builds. So
+        // we fake the rustc version and install from scratch every time since we can't detect
+        // the already-installed rustc version.
```

Comment:
The + thing sounds like something we could fix upstream in rustwide.

Reply:
This sounds like this is not blocking merging this PR, is it?

Reply:
( For merging I need to resolve the conversation :) )

Reply:
No, not a.blocker. would be nice to file an issue before it gets resolved and we lose the comment forever though

Reply:
kk, I don't know enought about this to create the issue, so I'm waiting for one of you two to do  it, and then also merge this PR.

Reply:
https://github.com/rust-lang/rustwide/issues/75